CONTRIBUTING.md

```diff
@@ -176,9 +179,10 @@ on their a11y-tree membership makes no difference. The queries where this does
 make a difference explicitly include this check e.g. `getByRole('button', { hidden: false })` (see [byRole documentation](https://testing-library.com/docs/dom-testing-library/api-queries#byrole) for more information).
 To see if your test (`test:browser` or `test:unit`) behaves the same between CI and local environment set the environment variable `CI` to `'true'`.
 
-### Testing the documentation site
+### Testing the changes on the documentation site
 
 The documentation site is built with Material-UI and contains examples of all the components.
+This is the best place to test your changes.
```

Comment:
Not a fan of this recommendation. You still have to write a test for the PR to be accepted. You might as well start with it. Docs are either an escape hatch if you don't know why they aren't working or if the change is visual only.

Reply:
What if we write something like this?

```
This is a good place to manually test your changes.
Automated tests for the pull requests to be approved may still be necessary.
```

Reply:
```suggestion
This is a great place to experiment with your changes.
```